docs/group_faqs/gf_multiple_account.rst

```diff
@@ -0,0 +1,31 @@
+Solace Cloud Multiple Account
+=============================
+
+Overview
+~~~~~~~~
+Users can be added to more than one Solace Cloud Account. Once added, a user can have different access and permission in each environment.
+
+For example, an organization may purchase two different subscriptions for (production, non-production, etc.) environments. The organization will be able to add users to one or more environments, enabling them to segregate their user base e.g. contractors, sales engineers, and different profession service groups.
+
+Access and Permissions
+~~~~~~~~~~~~~~~~~~~~~~
+The Multiple Account feature allows organizations to provide to their users’ different access and permissions to sets of different Solace Cloud PubSub+ Messaging Services. In addition, users can have different roles (e.g. admin, billing admin, and messaging user) in separate accounts.
+
+User with multiple account
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Login
+~~~~~
+The same credentials are used to login to all accounts in Solace PubSub+ Cloud Console. After successful login, a user that belongs to more than one account is prompted to select which account to be login to. If the user belongs to only one account, it will be directed to the console.
```

Comment:
I'm not sure the first sentence is needed, as the second sentence already makes it clear that the account selection is done after log in. 

"select which account to be login to" should be "select which account to log in to" grammatically, and I think "select which account to access" sounds better. 

"If the user belongs to only one account, it will be directed to the console." is unclear. Reword to something like "A user in only one account will not be prompted and will be signed in immediately."